runtime/conditions/merge.go

```diff
@@ -208,3 +216,12 @@ func stringInSlice(s []string, val string) bool {
 	}
 	return false
 }
+
+func generationInSlice(s []int64, val int64) bool {
```

Comment:
```suggestion
func int64InSlice(s []int64, val int64) bool {
```

Reply:
I had that before :smiley: but changed it to be more specific. Will make it `in64InSlice`.